templates/admin/failedJobs.tpl

```diff
@@ -32,25 +32,27 @@
 				</div>
 			</template>
 
-			<template slot-scope="{ row, rowIndex }">
+			<template #default="{ row, rowIndex }">
 				<table-cell
 					v-for="(column, columnIndex) in columns"
 					:key="column.name"
 					:column="column"
 					:row="row"
 					:tabindex="!rowIndex && !columnIndex ? 0 : -1"
 				>
-					<button-row v-if="column.name === 'actions'">
-						<pkp-button @click="redispatch(row)">
-							{translate key="admin.jobs.failed.action.redispatch"}
-						</pkp-button>
-						<pkp-button is-warnable @click="remove(row)">
-							{translate key="common.delete"}
-						</pkp-button>
-						<pkp-button element="a" is-link :href="row._hrefs._details">
-							{translate key="common.details"}
-						</pkp-button>
-					</button-row>
+					<template #default v-if="column.name === 'actions'">
```

Comment:
This is nuanced, but important. If its not wrapped in template - table-cell is having hard time whether something is passed to the slot. Because even if the v-if is false in previous implementation it still receives some abstract v-if element to the slot.